src/main/java/ti4/generator/TileGenerator.java

```diff
@@ -86,8 +85,8 @@ public TileGenerator(Game game, @Nullable GenericInteractionCreateEvent event, @
 
     private boolean isFowModeActive() {
         return game.isFowMode() && event != null &&
-                (event.getMessageChannel().getName().endsWith(Constants.PRIVATE_CHANNEL) ||
-                        event instanceof ShowGameAsPlayer.SlashCommandCustomUserWrapper);
+            (event.getMessageChannel().getName().endsWith(Constants.PRIVATE_CHANNEL) ||
```

Comment:
This class moved to `src/main/java/ti4/image`, you should be able to move both these classes over, take your changes (if it has merge conflicts), then compare with master to make sure no unintended changes were missed.